GitUI/CommandsDialogs/FormRebase.cs

```diff
@@ -182,6 +182,15 @@ private void InteractiveRebaseClick(object sender, EventArgs e)
             chkAutosquash.Enabled = chkInteractive.Checked;
         }
 
+        private void DateClick(object sender, EventArgs e)
+        {
+            chkCommitterDateIsAuthorDate.Enabled = !chkIgnoreDate.Checked;
+            chkIgnoreDate.Enabled = !chkCommitterDateIsAuthorDate.Checked;
+            chkInteractive.Enabled = !chkIgnoreDate.Checked && !chkCommitterDateIsAuthorDate.Checked;
+            chkPreserveMerges.Enabled = !chkIgnoreDate.Checked && !chkCommitterDateIsAuthorDate.Checked;
+            chkAutosquash.Enabled = chkInteractive.Checked && !chkIgnoreDate.Checked && !chkCommitterDateIsAuthorDate.Checked;
+        }
```

Comment:
I'd like to see a test for this functionality since it embodies the git command logic, which is defined elsewhere. 

Please follow the `TestAccessor` pattern (search the codebase for it), expose these UI elements through it and have tests something like:
```cs
[Test]
public void Check_ignore_date_should_toggle_non_applicable_settings()
{
    CheckBox chkIgnoreDate = form.GetTestAccessor().chkIgnoreDate;
    chkIgnoreDate.Checked.Should().BeFalse();

    chkIgnoreDate.Checked = true;

    chkIgnoreDate.Enabled.Should().BeFalse();
    // etc.

    chkIgnoreDate.Checked = false;

    chkIgnoreDate.Enabled.Should().BeTrue();
    // etc.
}
```